src/api/VoteMonitor.Entities/IOrderedEntity.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace VoteMonitor.Entities
+{
+    public interface IOrderedEntity
```

Comment:
Too bad C# does not understand structural type like TypeScript does 😀. I guess using reflection in the hierarchical mapper is too much to get rid of all these marker interfaces.

Reply:
I thought about using reflection but it sounded like an overhead for this case. I added these interfaces as they will be helpful in the future too in order to create generic implementations.

Reply:
Yup, I can see some use cases already